kurobako_core/src/hypervolume.rs

```diff
@@ -11,11 +11,11 @@ pub fn compute(pts: &[Vec<f64>], ref_pt: &[f64]) -> f64 {
 
 fn get_hypervolume_recursive(pts: &[Vec<f64>], ref_pt: &[f64]) -> f64 {
     match pts.len() {
-        1 => get_hypervolume_two_points(&pts[0], &ref_pt),
+        1 => get_hypervolume_two_points(&pts[0], ref_pt),
```

Comment:
For the record, a needless borrow. https://github.com/optuna/kurobako/runs/2901652078?check_suite_focus=true#step:5:110